schema.graphql

```diff
@@ -662,117 +669,19 @@ type Disbursed @entity {
     timestamp: BigInt!
 }
 
-# Trader Historical Data
-
-type TraderDayData @entity {
-    "format: <trader id>-<day id>"
-    id: ID!
-    trader: Trader!
-    "timestamp"
-    date: BigInt!
-    "trading volume aggregated per day for trader"
-    tradingVolume: BigDecimal!
-    "trading fee aggregated per day for trader"
-    fee: BigInt!
-    "total pnl aggregated per day for trader"
-    realizedPnl: BigInt!
-}
-
 # Referral
 
-# For tracking day trade metrics for referral codes on a per trader basis
-type ReferralCodeTraderDayData @entity {
-    "format: <referal code>-<day id>-<trader>"
-    id: ID!
-    "relationship to trader. since 2022-08-14, for contract traders, this value is changed from the tx sender to contract. eoa traders have no change"
-    trader: Trader!
-    "relationship to referral day data"
-    referralCodeDayData: ReferralCodeDayData!
-    "trading volume in notional value"
-    tradingVolume: BigDecimal!
-    "fees in notional value"
-    fees: BigDecimal!
-}
-
-type ReferralCodeDayData @entity {
```

Comment:
Just checked with Dylan. The frontend is still using this schema for vePERP referral. It can not be removed.